server/pairing/raw_message_handler.go

```diff
@@ -38,18 +35,19 @@ func (s *SyncRawMessageHandler) CollectInstallationData(rawMessageCollector *Raw
 	return err
 }
 
-func (s *SyncRawMessageHandler) PrepareRawMessage(keyUID, deviceType string) ([]byte, error) {
+func (s *SyncRawMessageHandler) PrepareRawMessage(keyUID, deviceType string) (rm []*protobuf.RawMessage, as []*accounts.Account, ss *settings.Settings, err error) {
+	ss = new(settings.Settings)
```

Comment:
Regarding the abbreviation of variable names, I'd like to share my thoughts. If the abbreviated variable is only used in a small scope, such as within 5 lines of code, I can accept it. However, if it is used within a 50-line function, I believe it increases the complexity of reading the code. 🙂

Reply:
My previous expression may not have been clear enough. I can accept abbreviated variable names, but it is better if abbreviated variable names only appear within a range of 5 lines of code. When abbreviated variable names appear within functions with up to 50 lines, it makes it a little difficult for me to read the code. The abbreviated variable names I referred e.g.: rm []*protobuf.RawMessage, as []*accounts.Account, ss *settings.Settings. I prefer to use rawMessage->rm, account->as, and setting->ss. If 'settings' collides with something, I will use naming such as 'setting' or 'targetSetting' or something else but rather than abbreviated `ss`. 🙂 (I knew you prefer pretty short names rm/as/ss 😉)

Reply:
That is completely fine. Let me change the variable to `syncSettings`, that way it is explicit and the collision is avoided.